kubernetes/namespaces/monitoring/alerts/alerts.d/nodes.yaml

```diff
@@ -41,9 +41,18 @@ groups:
 
   - alert: KubernetesNodeReady
     expr: kube_node_status_condition{condition="Ready",status="true"} == 0
-    for: 5m
+    for: 1m
     labels:
       severity: page
     annotations:
       summary: Kubernetes node ({{ $labels.kubernetes_node }} ) is marked as unready
       description: "Node {{ $labels.kubernetes_node }} has been unready for a long time"
+
+  - alert: KubernetesNodeCordoned
+    expr: kube_node_spec_unschedulable == 1
+    for: 1m
+    labels:
+      severity: page
+    annotations:
+      summary: Kubernetes node ({{ $labels.kubernetes_node }}) is cordoned
+      description: "Node {{ $labels.kubernetes_node }} has been cordoned"
```

Comment:
Why is this an alert? Isn't it a manual action? If it's a manual action, shouldn't we bump up the `for` time to something like 1-2 hours to leave it as a "in case you forgot it" alert?

Reply:
Good point, I'll up it to 30 minutes. In theory, a node could become "unschedulable" through something that we don't do, so I think having it somewhat low is a good idea in case something else happens, but I'll up it to 30 minutes.